csm/core/services/security.py

```diff
@@ -311,16 +311,24 @@ async def _timer_task(self, handler, start: datetime, interval: timedelta):
             current = datetime.now(timezone.utc)
             start = start + interval
 
+    async def _local_timezone(self, from_time):
+        """
+        Convert utc aware time to local timezone and print in specific format
+        """
+        local_time = from_time.astimezone()
```

Comment:
this is not the local time.. this is converted time with  timezone. Please rename variable accordingly

Reply:
renamed variable